### ISSUE TITLE
Continuous playback of voice messages is broken

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2279]
I am using Unigram 7.5 (5971) X64, in Windows 10 Version 20H2 (compilación de SO 19042.867).

When playing a voice message (tested in two groups and a private chat), the message appears to stop, according to the slider when the chat is minimiced at 95, 98 or 99%. Also, the "pause" button doesn't switch to "play", and the next message is not played. Thus continuous playback is broken in this Unigram version.

So, basically:

1. Play a voice message which below it has another voice message somewhere.

Expected: The next message is played.

Actual: It seems that the message doesn't finish playing (at least according to what's on screen), and the next message is not played.

**Comment by MarcAnt01** [comment]
Yep, even with a chat with a single message, it remains unread, because it doesn't finish to play

**Comment by cewbdex** [comment]
Also using the seek slider stops the playback completely.

**Comment by MarcAnt01** [comment]
> Also using the seek slider stops the playback completely.

I covered that in #2278

**Comment by FrayxRulez** [comment]
Should be fixed in https://github.com/UnigramDev/Unigram/commit/c9b4242cf039742f6a67b3e6f95579ed9d6c83e0
